### PR TITLE
Don't crash when no commands are passed

### PR DESCRIPTION
**Pull request by matanlurey**
Fixes https://github.com/dart-lang/build/issues/776.

See also https://github.com/dart-lang/build/issues/778 for potential future work.

### CODE REVIEW

File: build_runner/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+- **BUG FIX**: Running the `build_runner` binary without arguments no longer
```
Comment (jakemac53) [inline build_runner/CHANGELOG.md]: Nit: please add `## 0.7.1+1`

Comment (jakemac53) [reply build_runner/CHANGELOG.md]: Oh and update pubspec please :)

I think this is probly worth a publish right away

Comment (matanlurey) [reply build_runner/CHANGELOG.md]: Done.

**Comment by natebosch** [comment]
What is the new behavior if you run it with no args?

**Comment by matanlurey** [comment]
It just shows the command usage, but it also ensures the build script is ready.

**Pull request merged**